tests/parts/test_slide.py

```diff
@@ -795,3 +814,50 @@ def SlideMaster_(self, request, slide_master_):
     @pytest.fixture
     def slide_master_(self, request):
         return instance_mock(request, SlideMaster)
+
+    @pytest.fixture
+    def theme_part_(self, request):
+        return instance_mock(request, ThemePart)
+    
+    @pytest.fixture
+    def part_related_by_(self, request):
+        return method_mock(request, SlideMasterPart, "part_related_by", autospec=True)
+
+
+class DescribeThemePart(object):
+    def it_knows_its_name(self, name_fixture):
+        theme, expected_value = name_fixture
+        assert theme.name == expected_value
+
+    def it_provides_access_to_its_theme(self, theme_fixture):
+        theme_part, Theme_, sldTheme, theme_ = theme_fixture
+        theme = theme_part.theme
+        Theme_.assert_called_once_with(sldTheme)
+        assert theme is theme_
+
+
+    # fixtures -------------------------------------------------------
+
+    @pytest.fixture
+    def theme_fixture(self, Theme_, theme_):
+        sldTheme = element("a:theme")
+        theme_part = ThemePart(None, None, sldTheme)
+        return theme_part, Theme_, sldTheme, theme_
+
+    @pytest.fixture
+    def name_fixture(self):
+        theme_cxml, expected_value = "a:theme{name=Foobar}", "Foobar"
+        theme = element(theme_cxml)
+        theme = Theme(theme)
+        return theme, expected_value
```

Comment:
This is the same as the `name_fixture` on `test_theme.py` on line 53, can we move this to a separate file where they can both import it?

Reply:
Like I said in my other comment, at the expense of a bit of repeated code, this codebase keeps all the fixtures required in self contained classes so it is really simple and easy to follow.